install_rustup.sh

```diff
@@ -15,28 +16,58 @@ global_azle_target_dir=$global_azle_config_dir/target
 export CARGO_HOME=$global_azle_config_dir
 export RUSTUP_HOME=$global_azle_config_dir
 
+function install_wasm32_unknown_unknown() {
+    $global_azle_rustup_bin target add wasm32-unknown-unknown &> $global_azle_logs_dir/step_2_wasm32_unknown_unknown
```

Comment:
The VS Code extension [ShellCheck](https://marketplace.visualstudio.com/items?itemName=timonwong.shellcheck) recommends double quoting when using variables to prevent globbing and word splitting.

I think that means:

```suggestion
    $global_azle_rustup_bin target add wasm32-unknown-unknown &> "$global_azle_logs_dir/step_2_wasm32_unknown_unknown"
```

This would apply anywhere you're using a variable I think.